resources/dicts/relationship_events/normal_interactions/dislike/increase.json

```diff
@@ -78,23 +157,53 @@
 			"r_c is asking m_c to tell {PRONOUN/r_c/object} about how good {PRONOUN/r_c/subject} {VERB/m_c/look/looks}.",
 			"r_c offends m_c with {PRONOUN/r_c/poss} brutal honesty.",
 			"r_c is boasting about {PRONOUN/r_c/poss} accomplishments to m_c, but m_c isn't impressed.",
-			"r_c gets really close to m_c's face while talking and grosses m_c out with {PRONOUN/r_c/poss} bad breath."
+			"r_c gets really close to m_c's face while talking and grosses m_c out with {PRONOUN/r_c/poss} bad breath.",
+			"r_c doesn't understand why m_c lied to save a cats feelings."
```

Comment:
```suggestion
			"r_c doesn't understand why m_c lied to save a cat's feelings."
```

Reply:
Looked it up, it needed to be cats' actually so fixed